fdbclient/BlobGranuleFiles.cpp

```diff
@@ -357,7 +357,7 @@ struct IndexBlockRef {
 			// Compressing indexBlock will need offset recalculation (circular depedency). IndexBlock size is bounded by
 			// number of chunks and sizeof(KeyPrefix), 'not' compressing IndexBlock shouldn't cause significant file
 			// size bloat.
-
+			CODE_PROBE(true, "encrypting index block");
```

Comment:
can you please add the corresponding probes for decryption?

Reply:
Decryption already has code probes